front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -13,19 +13,18 @@ type Option = {
   unit?: string;
```

Comment:
I think that options never have a unit, can you remove this ?

Reply:
Yes, you're right. It is only needed in the selectedUnit state.

Reply:
Yes I think we can remove it now